src/main/java/Parser.java

```diff
@@ -0,0 +1,85 @@
+public class Parser {
+    public static void addOrCompleteTask(String input, Task[] tasks){
```

Comment:
This method exceeds 30 LOC and hence violates the code quality guideline on "Avoid Long Methods".

Reply:
Thanks!